src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateConstructorFromMembers/GenerateConstructorFromMembersTests.cs

```diff
@@ -627,5 +629,151 @@ public Contribution(string title, int number)
 }",
 options: Option(CodeStyleOptions.QualifyFieldAccess, CodeStyleOptions.TrueWithSuggestionEnforcement));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateConstructorFromMembers)]
+        public async Task TestSingleFieldWithDialog()
+        {
+            await TestWithGenerateConstructorDialogAsync(
+@"using System.Collections.Generic;
+
+class Z
+{
+    int a;
+    [||]
```

Comment:
Nope.  Only on completely blank lines between members.  Which i think is reasonable. :)